packages/yuudachi/src/functions/reports/utils.ts

```diff
@@ -0,0 +1,7 @@
+export function reportRedisMessageKey(guildId: string, userId: string) {
+	return `guild:${guildId}:report:messages:${userId}`;
+}
+
+export function reportRedisUserKey(guildId: string, userId: string) {
+	return `guild:${guildId}:report:user:${userId}`;
+}
```

Comment:
Cool idea, but lets not do this yet.

Later we can have a utility function that generates the keys in one place for all the ones we have 👍

Reply:
Reverted in fd64a47